index.html

```diff
@@ -373,9 +374,9 @@ <h4>
                         </div>
                     </div>
                     <div class="container">
-                        <h2 class="text-center mb-5 border-bottom-3">Our grateful students</h2>
+                        <h2 class="text-center mb-5 border-bottom-3 section-header">Our grateful students</h2>
                     </div>
-                    <div class=" card-deck">
+                    <div class="card-deck">
                         <div class="card">
                             <img src="https://images.unsplash.com/photo-1507120878965-54b2d3939100?ixlib=rb-0.3.5&q=80&fm=jpg&crop=faces&fit=crop&h=200&w=200&s=99fbace66d1bfa48c9c6dc8afcac3aab"
```

Comment:
There should behave alt for images when they are not displayed. Example: alt="face_image"